src/main/java/newbank/server/Customer.java

```diff
@@ -3,9 +3,15 @@
 import java.util.ArrayList;
 
 public class Customer {
-	
+	private String username;
+	private String password;
 	private ArrayList<Account> accounts;
 	
+	public Customer( final String username, final String password ) {
```

Comment:
These parenthesis seem inconsistent with the rest of the code.

Reply:
Correct, fixed.